src/benchmark/operators/join_benchmark.cpp

```diff
@@ -20,9 +20,6 @@ constexpr auto NUMBER_OF_CHUNKS = size_t{50};
 constexpr auto TABLE_SIZE_SMALL = size_t{1'000};
 constexpr auto TABLE_SIZE_MEDIUM = size_t{100'000};
 constexpr auto TABLE_SIZE_BIG = size_t{10'000'000};
-}  // namespace
-
-namespace opossum {
```

Comment:
Why moving out of the namespace?

Reply:
Everything that is in an anonymous namespace can't inadvertently leak into others